kernel/packages/shared/apis/SceneStateStorageController/SceneStateStorageController.ts

```diff
@@ -91,9 +91,26 @@ export class SceneStateStorageController extends ExposableAPI implements ISceneS
   }
 
   @exposeMethod
-  async publishSceneState(sceneId: string, sceneState: SerializedSceneState): Promise<DeploymentResult> {
+  async publishSceneState(sceneId: string, sceneName: string, sceneDescription: string, sceneState: SerializedSceneState): Promise<DeploymentResult> {
```

Comment:
I want to clarify that changes in signatures like this one are only enabled before we make the public release, after that moment, we should create a new function accepting new arguments, to not break backwards compatibility

Reply:
to clarify just in case, we are good now, but watch out for the future changes

Reply:
Yeah, you're right. At the moment, as we are in a "pre-launch" stage, we are still designing certains parts. Thanks for the clarifications!